### PR TITLE
DOC: stats.skewtest: add semi-realistic example

### PR DESCRIPTION
**Pull request by mdhaber**
#### What does this implement/fix?
This adds a semi-realistic example to the documentation of `scipy.stats.skewtest`.

### CODE REVIEW

File: scipy/stats/_stats_py.py [hunk 1]
```diff
@@ -1911,14 +2001,11 @@ def jarque_bera(x, *, axis=None):
     ...     return x.shape[axis]/6 * (s**2 + k**2/4)
     >>> res = stats.monte_carlo_test(x, stats.norm.rvs, statistic)
     >>> fig, ax = plt.subplots(figsize=(8, 5))
-    >>> ax.plot(jb_val, pdf)
+    >>> jb_plot(ax)
```
Comment (mdhaber) [inline scipy/stats/_stats_py.py:2004]: This is an unrelated simplification I thought I'd sneak in here.

File: scipy/stats/_stats_py.py [hunk 2]
```diff
@@ -1545,22 +1545,112 @@ def skewtest(a, axis=0, nan_policy='propagate', alternative='two-sided'):
...
+    - The inverse is not true; that is, the test is not used to provide
+      evidence for the null hypothesis.
+    - The threshold for values that will be considered "small" is a choice that
+      should be made before the data is analyzed [3]_ with consideration of the
+      risks of both false positives (incorrectly rejecting the null hypothesis)
+      and false negatives (failure to reject a false null hypothesis).
+
+    Note that the standard normal distribution provides an asymptotic
+    approximation of the null distribution; it is only accurate for samples
+    with many observations. For small samples like ours,
```
Comment (mdhaber) [inline scipy/stats/_stats_py.py:1634]:
```suggestion
    with sufficiently many observations. For small samples like ours,
```

**Review by tupui — APPROVED** [review]
Good example and all renders properly. I don't have anything to say here (and the plotting code was reviewed in a previous PR). CI is green, merging. Thanks Matt!
